packages/td-tools/src/td-parser.ts

```diff
@@ -222,20 +222,20 @@ export function serializeTD(thing: Thing): string {
     const copy = JSON.parse(JSON.stringify(thing));
 
     // clean-ups
-    if (!copy.security || copy.security.length === 0) {
+    if (copy.security == null || copy.security.length === 0) {
         copy.securityDefinitions = {
             nosec_sc: { scheme: "nosec" },
         };
         copy.security = ["nosec_sc"];
     }
 
-    if (copy.forms && copy.forms.length === 0) {
+    if (copy?.forms.length === 0) {
```

Comment:
After also testing locally, I now found out that the real issue is in this line :)

```suggestion
    if (copy.forms?.length === 0) {
```

Reply:
Yes, that makes sense! Sorry for not seeing it myself.. I still wonder why it works locally on my side!?